trusted-committer/03-keeping-the-community-healthy.asciidoc

```diff
@@ -1,80 +1,76 @@
-good reason not to. Trusted Committers do this because they understand the virtuous cycle:
-good mentoring in pull requests leads to a higher level of trust and engagement on part
-of the https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] which in turn leads to more people willing to make
-contributions and thus more contributions. We'll discuss more about this in the
-segment in https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/trusted-committer/04-uplevelling-community-members.asciidoc["Upleveling the Community"].
+good reason not to. Good mentoring during pull requests leads to a higher level
+of trust and engagement by https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_], which in turn leads
+to more contributions. We’ll discuss this more in pass:[<a data-type="xref" href="upleveling" data-xrefstyle="chap-num-title">#upleveling</a>].
 
-Finally, some people stick around in InnerSource communities because they get
-to focus on developing software and to spend as little time as possible on
-activities which are considered overhead or waste, common especially in large
-companies with a strong focus on processes.  The Trusted Committer's job in this context is to
-ensure that https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] can actually do that and generally to keep things
-running smoothly. This includes communicating and enacting helpful
-contribution guidelines.
+Finally, some people stick around in InnerSource communities because
+they get to focus on developing software instead of activities considered overhead or waste, especially 
+common in large companies with a strong focus on processes. The Trusted Committer's job in this context is to
+ensure that https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] can actually focus on their projects by
+communicating and enacting helpful contribution guidelines.
 
-One important aspect of these is to explain what I
-call _signaling_ in pull requests: what should a comment look like? What does
-it mean if I _like_ or _+1_ a comment?  How is @mentioning someone with a /CC
-prefix different from doing so with a /FYI prefix? Generally speaking, Trusted Committers need
-to make sure that the contribution process uncovers problems, rather than being
-the cause of them.  Ultimately, Trusted Committers should empower their community to point out
-process related problems and to adapt and improve them as a community as much
-as possible.
+One important aspect of these guidelines is to explain what we call _signaling_ in
+pull requests: what should a comment look like? What does it mean if I
```

Comment:
Should the first what not be capitalized too?

Reply:
@lenucksi , I do not see any issues with capitalization in this sentence. If you mean "what" in "what should a comment look like?", the answer is no - as written, the "what" after the colon should begin with lower case. Please let me know if you meant something else.
Thank you!

Reply:
Could have a period instead of a colon and then it will be obvious to capitalize the `W`.